### DISCUSSION TITLE
Crashes on startup

### DISCUSSION THREAD

**Discussion by SlavaProvalov** [dbeaver/dbeaver#13731] [Q&A] [answered]
#### System information: 
- Windows 10 Pro x64 21H1
- DBeaver version: 21.1.5 (same problem on all previous versions after 2x.x.x)
- Java version - 11.0.11 

#### Describe the problem you're observing:
DBeaver crashes on startup. 
![dbeaver_starts](https://user-images.githubusercontent.com/16419946/131214864-b78a4abc-e38e-47c7-a0e2-c75e6da27cb0.jpg)

.log:

!SESSION 2021-08-28 13:07:55.398 -----------------------------------------------
eclipse.buildId=unknown
java.version=11.0.11
java.vendor=AdoptOpenJDK
BootLoader constants: OS=win32, ARCH=x86_64, WS=win32, NL=ru
Command-line arguments:  -os win32 -ws win32 -arch x86_64

!ENTRY org.eclipse.e4.ui.workbench 2 0 2021-08-28 13:07:59.673
!MESSAGE Removing PartDescriptorImpl with the "org.eclipse.gef.ui.palette_view" id and the "Palette" label.It points to the non available "bundleclass://org.eclipse.ui.workbench/org.eclipse.ui.internal.e4.compatibility.CompatibilityView" class. Bundle might have been uninstalled

!ENTRY org.eclipse.e4.ui.workbench 2 0 2021-08-28 13:07:59.673
!MESSAGE Removing PartDescriptorImpl with the "org.eclipse.ui.views.ProgressView" id and the "Фоновые задачи" label.It points to the non available "bundleclass://org.eclipse.ui.workbench/org.eclipse.ui.internal.e4.compatibility.CompatibilityView" class. Bundle might have been uninstalled

!ENTRY org.eclipse.e4.ui.workbench 2 0 2021-08-28 13:07:59.689
!MESSAGE Removing PartDescriptorImpl with the "org.eclipse.ui.texteditor.TemplatesView" id and the "Шаблоны" label.It points to the non available "bundleclass://org.eclipse.ui.workbench/org.eclipse.ui.internal.e4.compatibility.CompatibilityView" class. Bundle might have been uninstalled

!ENTRY org.eclipse.e4.ui.workbench 2 0 2021-08-28 13:07:59.689
!MESSAGE Removing PartDescriptorImpl with the "org.jkiss.dbeaver.tasks" id and the "Database Tasks" label.It points to the non available "bundleclass://org.eclipse.ui.workbench/org.eclipse.ui.internal.e4.compatibility.CompatibilityView" class. Bundle might have been uninstalled
[dbeaver-debug.log](https://github.com/dbeaver/dbeaver/files/7070549/dbeaver-debug.log)

**Comment by ShadelessFox** [comment]
Hello @SlavaProvalov,

Please provide more context. Is DBeaver installed as extension in Eclipse or as standalone application?

**Comment by Cyberuben** [comment]
I'm not sure if this is relevant, but I'm not able to start up the application at all. I just downloaded the latest version, tried to open it. I can see the `dbeaver.exe` in my task manager, but I can't seem to find any open window or use the application at all.

**Comment by SlavaProvalov** [comment]
> Hello @SlavaProvalov,
> 
> Please provide more context. Is DBeaver installed as extension in Eclipse or as standalone application?

As standalone application

**Comment by ShadelessFox** [comment]
Are you installing DBeaver through .msi installer of .zip archive?

**Comment by SlavaProvalov** [comment]
> Are you installing DBeaver through .msi installer of .zip archive?

Msi installer

**Comment by ShadelessFox** [comment, marked as answer]
Try resetting workspace:
https://github.com/dbeaver/dbeaver/wiki/Reset-workspace

If it didn't help, try uninstalling DBeaver first and repeat.

**Comment by SlavaProvalov** [comment]
> Try resetting workspace:
> https://github.com/dbeaver/dbeaver/wiki/Reset-workspace
> 
> If it didn't help, try uninstalling DBeaver first and repeat.

Thanks! Reseting workspace realy helped me.

**Reply by orca654** [reply]
I found if I pointed the DBeaver to another Java source, it would work. for example:
"C:\Program Files\DBeaverEE\dbeaver.exe" -nl en -vm "C:\ArcGIS\Pro\java\runtime\jre\bin\server"
